Dialog/SpritesEditorDialog.cpp

```diff
@@ -559,3 +679,116 @@ void SpritesEditorDialog::on_pushButton_SwapPal_clicked()
     SetSelectedSpriteTile(0);
     RenderSpritesetTileMapAndResetLoadTable();
 }
+
+/// <summary>
+/// Reset Oam designer with new OAM data string when clicking pushButton_ResetAllOamData
+/// </summary>
+void SpritesEditorDialog::on_pushButton_ResetAllOamData_clicked()
+{
+    QStringList List_strs = QInputDialog::getText(this, tr("InputBox"),
```

Comment:
let's use camelCase for the variable name

Reply:
done